cmd/podman/manifest_annotate.go

```diff
@@ -0,0 +1,166 @@
+package main
```

Comment:
this command will need to fenced off as a local-only command until we have apiv2 endpoints for this.  maybe that should get done now? so that this is a slightly cleaner implementation?